docs/source/AdministratorGuide/Systems/DataManagement/fts.rst

```diff
@@ -1,27 +1,154 @@
+
+
+FTSAgent
+--------
+
+This is the complex one. The agent is going to fetch the request in state `Scheduled` in the RMS, and look in the FTSDB for the associated FTSFiles. It is then going to monitor submitted jobs, submit new jobs with new files or files that failed previously, register files successfuly transfered
+
+The agent still supports old FTS2 server, but since there are no such servers anymore, this behavior will not be detailed here.
+
+
+Configuration options
+^^^^^^^^^^^^^^^^^^^^^
+
+
+  * FTSPlacementValidityPeriod: deprecated (FTS2)
+  * MaxActiveJobsPerRoute: deprecated (FTS2)
+  * MaxFilesPerJob: maximum number of files in a single fts job
+  * MaxRequests: maximum number of requests to look at per agent's cycle
+  * MaxThreads: maximum number of threads
+  * MaxTransferAttempts: maximum number of time we attempt to transfer a file
+  * MinThreads: minimum number of threads
+  * MonitorCommand: deprecated (FTS2)
+  * MonitoringInterval: interval between two monitoring of an FTSJob (in second)
+  * PinTime: when staging, pin time requested in the FTS job (in second)
+  * ProcessJobRequests: True if this agent is meant to process job only transfers (see `Multiple FTSAgents`_)
+  * SubmitCommand: deprecated (FTS2)
+
+
+
+
+
+File registration
+^^^^^^^^^^^^^^^^^
+
+The FTSAgent runs with the DataManagement shifter proxy, and hense can register the files directly after they have been transfered. If the registration fails, the FTSAgent still considers the transfer as done, and adds a RegisterFile operation in the RMS Request from which the transfers originated
+
+
+.. _multipleFTSAgents:
+
+Multiple FTSAgents
+^^^^^^^^^^^^^^^^^^
+
+It is not possible to have several FTSAgents running in parallel except in a very specific configuration, which is 1 agent taking care of the failover transfers, 1 agent taking care of the transformation transfers. This behavior is enabled by the `ProcessJobRequests` flag. But be careful, two agents taking care of the same case would lead to problems.
+
+Without entering the details on how to install several instances of the same agent, if you want such a configuration, it would look something like.
+
+.. code-block:: python
+
+   FTSAgent
+   {
+     # All the common options
+   }
+
+   FTSAgentTransformations
+   {
+     Module = FTSAgent
+     ProcessJobRequests = False
+     ControlDirectory = control/DataManagement/FTSAgentTransformations
+     # whatever other options
+     # ...
+   }
+
+   FTSAgentFailover
+   {
+     Module = FTSAgent
+     ProcessJobRequests = True
+     ControlDirectory = control/DataManagement/FTSAgentFailover
+     # whatever other options
+     # ...
+   }
+
+
+FTSServer policy
+^^^^^^^^^^^^^^^^
+
+The FTS server to which the job is sent is chose based on the policy. There are 3 possible policy:
+
+  * Random: the default. makes a random choice
+  * Failover: pick one, and stay on that one until it fails
+  * Sequence: take them in turn, always change
```

Comment:
For completeness, maybe refer to the dirac-dms-fts-* scripts? There are also .rst files with them.

Reply:
I have to check if they even do something !